### ISSUE TITLE
Make wolfy87-eventemitter an optional dependency

### ISSUE THREAD

**Issue by jankapunkt** [meteorrn/meteor-react-native#112]
As stated in the [code](https://github.com/meteorrn/meteor-react-native/blob/master/lib/socket.js#L2):

```
// TODO: 0.60+ supports import { EventEmitter } from 'events';
```

Therefore we might rather make this an optional dependency and update the docs. RN > 60 should be very common by now.

**Comment by TheRealNate** [comment]
@jankapunkt For compatibility [we already say](https://github.com/meteorrn/meteor-react-native#compatibility) that RN <0.60 isn't supported, so I don't think we even need to make it optional, we can just fully move over.

**Comment by jankapunkt** [comment]
keep open

**Comment by jankapunkt** [comment]
Implemented in 2.5.0

get it via https://www.npmjs.com/package/@meteorrn/core/v/2.5.0
